api/column_family.cc

```diff
@@ -220,14 +220,18 @@ static json::json_return_type sum_map(const std::unordered_map<sstring, uint64_t
     return res;
 }
 
+static sstring sstable_location(const sstables::shared_sstable& sst) {
+    return sst->get_storage().location(*sst);
```

Comment:
could just use the member function of `sstable`:
```c++
static sstring sstable_location(const sstables::shared_sstable& sst) {
  return sst->filename(component_type::Data);
}
```

Reply:
I don't want sstable to export filename() method at all. It could be `t->location()` though

Reply:
i see. should have read the title and the cover letter again..